lib/src/runner/configuration.dart

```diff
@@ -307,6 +322,8 @@ class Configuration {
             : Uri.parse("http://localhost:$pubServePort"),
         _concurrency = concurrency,
         _paths = _list(paths),
+        _exceptPackages = _list(exceptPackages),
+        _onlyPackages = _list(onlyPackages),
```

Comment:
We should also ensure this at the configuration level, so users of the API can make simplifying assumptions. Also, make sure to document that.